packages/fuel-indexer-lib/src/manifest.rs

```diff
@@ -46,6 +47,30 @@ impl ToString for Module {
     }
 }
 
+/// Represents contract IDs in a `Manifest` struct.
+///
+/// - `Single`: Represents an optional single contract ID as an `Option<String>`.
```

Comment:
Same comment as [here](https://github.com/FuelLabs/fuel-indexer/pull/755#discussion_r1165731050) with regard to Rust doc struct field comments